flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/Execution.java

```diff
@@ -131,6 +133,35 @@
 	
 	private SerializedValue<StateHandle<?>> operatorState;
 
+	/* Lock for updating the accumulators atomically. */
```

Comment:
Why not follow the style of the remaining class, with respect to empty lines between fields?